restservice/src/main/java/org/itevents/dao/mybatis/sql_session_dao/UserMyBatisDao.java

```diff
@@ -68,6 +70,30 @@ public void setUserPassword(User user, String password) {
         getSqlSession().update("org.itevents.dao.mybatis.mapper.UserMapper.setUserPassword", new UserPassword(user, password));
     }
 
+    @Override
+    public void setOtpToUser(User user, OtpGenerator otpGenerator) {
```

Comment:
Can we add `user` field to OTP and don't create additional class (`UserPassword`)?